src/Models/Context/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -160,7 +179,7 @@ const buildDeeplink = (
             : undefined,
         mode: currentState.mode,
         adtUrl: currentState.adtUrl,
-        storageUrl: currentState.storageUrl
+        storageContainerUrl: currentState.storageContainerUrl
```

Comment:
why is this change needed? The public contract should not need to change. THat's why it's decoupled from our internal contracts. Ideally this would not break any existing links that exist in the wild.

Reply:
"storageUrl" is vague since there is storage account url and storage blob container url - which are different concepts. Even if this is the case, do you think I should revert this since it is going to break the existing deeplinks? @msnyder-msft

Reply:
I think it's fine that the public url has the shorter version since they don't need to discriminate between those 2 concepts. I'd vote to keep it as `storageUrl` if for no other reason than it's a shorter url.

Reply:
reverted changes: https://github.com/microsoft/iot-cardboard-js/pull/684/commits/5f8c8f5c9997d7b5e99981cd825512530777a782